other/bootstrap_daemon/tox-bootstrapd.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
+ExecStartPre=-mkdir /var/run/tox-bootstrapd -p
+ExecStartPre=chown tox-bootstrapd:tox-bootstrapd -R /var/run/tox-bootstrapd
+WorkingDirectory=/var/lib/tox-bootstrapd
+ExecStart=/usr/bin/tox-bootstrapd /etc/tox-bootstrapd.conf
+User=tox-bootstrapd
+Group=tox-bootstrapd
+PIDFile=/var/run/tox-bootstrapd.pid
```

Comment:
Is tox-bootstrapd really the user you want it to run as? It won't be consistent with the other daemon installs.